presidio-image-redactor/Pipfile.lock

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
@niwilso the CI hanging could be because this was locked on an Apple silicone (Assuming that's the case). @SharonHart based on your observations, it is best to lock this using Linux and Python 3.8, right?